packages/terra-core-docs/src/terra-dev-site/doc/section-header/example/AccordionSectionHeader.jsx

```diff
@@ -9,13 +9,15 @@ const AccordionSectionHeader = () => {
   };
 
   const sectionHeaderProps = {
-    title: 'I can accordion, click me',
     isOpen,
     onClick: handleClick,
   };
 
   return (
-    <SectionHeaderExampleTemplate title="Accordion Section Header" exampleProps={sectionHeaderProps} />
+    <AccoordianExampleTemplate title="I can accordion, click me" heading="Accordion Section Header" exampleProps={sectionHeaderProps}>
```

Comment:
```suggestion
    <AccoordianExampleTemplate title="I can accordion, click me" heading="Accordion Section Header" sectionHeaderAttrs={sectionHeaderProps}>
```
Nit : this will remove the need of line no 82 in AccordionExampleTemplate.jsx
`const sectionHeaderProps = { ...sectionHeaderAttrs };`